compiler/passes/normalize.cpp

```diff
@@ -1507,15 +1500,15 @@ fixup_query_formals(FnSymbol* fn) {
         std::vector<SymExpr*> symExprs;
         collectSymExprs(fn, symExprs);
         if (call->isNamed("_build_tuple")) {
-          add_to_where_clause(formal, new SymExpr(new_IntSymbol(call->numActuals())), new CallExpr(PRIM_QUERY, new_StringSymbol("size")));
+          add_to_where_clause(formal, new SymExpr(new_IntSymbol(call->numActuals())), new CallExpr(PRIM_QUERY, new_CStringSymbol("size")));
           call->baseExpr->replace(new SymExpr(dtTuple->symbol));
           isTupleType = true;
         }
         CallExpr* positionQueryTemplate = new CallExpr(PRIM_QUERY);
         for_actuals(actual, call) {
           if (NamedExpr* named = toNamedExpr(actual)) {
-            positionQueryTemplate->insertAtTail(new_StringSymbol(named->name));
-            CallExpr* nameQuery = new CallExpr(PRIM_QUERY, new_StringSymbol(named->name));
+            positionQueryTemplate->insertAtTail(new_CStringSymbol(named->name));
+            CallExpr* nameQuery = new CallExpr(PRIM_QUERY, new_CStringSymbol(named->name));
             if (DefExpr* def = toDefExpr(named->actual)) {
               replace_query_uses(formal, def, nameQuery, symExprs);
             } else {
```

Comment:
Also in normalize in init_untyped_var, there is some code to promote initializations of variables with c string literal type in to full strings, to so that in this case:
 var x = c"mystring";

x would have type string rather than cString.

I don't think we should keep this code, but wanted to ask you about it.
Also, are we automatically promoting c string literals to other strings anywhere? (maybe I havn't gotten to it yet).